### PR TITLE
Added Description to CancellationPolicy

### PR DESCRIPTION
**Pull request by AlexeyShok**
Added Description to CancellationPolicy

### CODE REVIEW

File: HappyTravel.EdoContracts/Accommodations/Internals/CancellationPolicy.cs [hunk 1]
```diff
@@ -25,13 +26,19 @@ public CancellationPolicy(DateTime fromDate, double percentage)
...
 
+        /// <summary>
+        ///     The policy description. 
+        /// </summary>
+        public string? Description { get; }
+
 
-        public bool Equals(in CancellationPolicy other) => FromDate.Equals(other.FromDate) && Percentage.Equals(other.Percentage);
+        public bool Equals(in CancellationPolicy other) 
+            => FromDate.Equals(other.FromDate) && Percentage.Equals(other.Percentage) && Description.Equals(other.Description);
```
Comment (mingazhev) [inline HappyTravel.EdoContracts/Accommodations/Internals/CancellationPolicy.cs:36]: Possible NRE there. You can use `string.Equals` static method instead

Comment (AlexeyShok) [reply HappyTravel.EdoContracts/Accommodations/Internals/CancellationPolicy.cs]: Fixed

**Review by mingazhev — APPROVED** [review]

File: HappyTravel.EdoContracts/Accommodations/Internals/CancellationPolicy.cs [hunk 2]
```diff
@@ -25,13 +26,19 @@ public CancellationPolicy(DateTime fromDate, double percentage)
         /// </summary>
         public double Percentage { get; }
 
+        /// <summary>
+        ///     The policy description. 
+        /// </summary>
+        public string? Description { get; }
```
Comment (kirillta) [inline HappyTravel.EdoContracts/Accommodations/Internals/CancellationPolicy.cs:32]: I think, it should call `Remark` like in another similar places.

**Review by kirillta — COMMENTED** [review]
We have `Deadline.Remarks` property one level above a policy. Should we still keep it?
@mingazhev what do you think?

File: HappyTravel.EdoContracts/Accommodations/Internals/CancellationPolicy.cs (hunk 2 above)
Comment (AlexeyShok) [reply HappyTravel.EdoContracts/Accommodations/Internals/CancellationPolicy.cs]: Renamed

**Review by kirillta — APPROVED** [review]

**Pull request merged**